miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SampleController.java

```diff
@@ -280,4 +280,11 @@ public SampleDto updateSample(@PathVariable("id") Long id, @RequestBody SampleDt
     }
     return uniqueIdentities.stream().map(Dtos::asDto).collect(Collectors.toSet());
   }
+
+  @RequestMapping(value = "/sample/query", method = RequestMethod.POST, produces = { "application/json" })
+  @ResponseBody
+  public List<SampleDto> getSamplesInBulk(@RequestBody List<String> names, HttpServletRequest request, HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)